### DISCUSSION TITLE
Problems with filament installation

### DISCUSSION THREAD

**Discussion by nurmuhamad45** [filamentphp/filament#13085] [Help]
### Package

Other

### Package Version

v3.2

### How can we help you?

`composer require filament/filament:"^3.2"`

Your requirements could not be resolved to an installable set of packages.

  Problem 1
    _- filament/support v3.2.0 requires ext-intl * -> it is missing from your system. Install or enable PHP's intl extension.
    - filament/filament v3.2.0 requires filament/support v3.2.0 -> satisfiable by filament/support[v3.2.0].
    - Root composer.json requires filament/filament 3.2 -> satisfiable by filament/filament[v3.2.0]._

To enable extensions, verify that they are enabled in your .ini files:
    - C:\php\php.ini
You can also run `php --ini` in a terminal to see which files are used by PHP in CLI mode.
Alternatively, you can run Composer with `--ignore-platform-req=ext-intl` to temporarily ignore these required extensions.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

**Comment by leandrocfe** [comment]
- install `ext-intl`
- restart the webserver (or reboot the system)
- run `composer require filament/filament -W`
